### DISCUSSION TITLE
testnet faucet issue

### DISCUSSION THREAD

**Discussion by shrivrat0809** [Cyfrin/foundry-full-course-cu#237] [Q&A] [answered]
my chainlink sepolia faucet is not giving me testnet eth it is just showing me "send 20 token linnk". please help!!!

**Comment by shikhar229169** [comment]
Have you verified the captcha?

**Comment by hardik201729** [comment]
This problem occurs to me also. So this time use this link instead "https://sepoliafaucet.com/". You can get 0.5 sepoliaETH from here.

**Reply by 0xusmanf** [reply, marked as answer]
Hello, @shrivrat0809! As I can guess from your question you are trying to request Sepolia ETH from the LINK token faucet which is [https://faucets.chain.link/](https://faucets.chain.link/), here you can only request LINK tokens. To request Sepolia ETH you have to go to the following link [https://sepoliafaucet.com/](https://sepoliafaucet.com/) as mentioned by @hardik201729 and you have to sign in there using your Alchemy account before requesting Sepolia ETH.
